### PR TITLE
Fill empty space in two sided

### PR DESCRIPTION
**Pull request by SiAdcock**

### CODE REVIEW

File: src/css/embed.scss [hunk 1]
```diff
@@ -197,25 +212,12 @@ $brexit-control-button-size: 28px;
...
-    @include fs-textSans(3);
-    font-weight: 900;
-    .brexit__nav-control__label {
-        padding: 0 $gutter;
-        width: auto;
-    }
-}
-
 .brexit__nav-control__label {
+    @include button($brexit-control-button-size);
```
Comment (joelochlann) [inline src/css/embed.scss:216]: as a separate PR we can look at refactoring to use this `button()` mixin more widely

File: src/css/embed.scss [hunk 2]
```diff
@@ -252,31 +254,127 @@ $brexit-control-button-size: 28px;
...
-    float: left;
-    // currently hard-coded to 3 sliders
-    // TODO: variable number of sliders
-    width: 33.3%;
+    flex-grow: 1;
+    flex-basis: 0;
+    margin: 0 $brexit-control-button-size*0.5;
+    padding: 6px 21px;
+    background-color: colour(news-main-1);
+    width: 1px;
```
Comment (joelochlann) [inline src/css/embed.scss]: oh, let's put a comment here about why we've done this!

File: src/css/embed.scss [hunk 3]
```diff
@@ -252,31 +254,127 @@ $brexit-control-button-size: 28px;
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .brexit__header {
+        padding-bottom: 0;
+    }
+}
+.brexit__two-sided__top, .brexit__two-sided__bottom {
+    display: flex;
```
Comment (joelochlann) [inline src/css/embed.scss:322]: also for a separate PR, but let's look at refactoring to pull out some of these common flex-box setups into mixins

**Comment by joelochlann** [comment]
👍  Looks good! A few tidy-ups we can do in other PRs - plus we need to put a comment explaining that 1px width thing

File: src/css/embed.scss (hunk 1 above)
Comment (SiAdcock) [inline src/css/embed.scss:216]: 👍

File: src/css/embed.scss (hunk 3 above)
Comment (SiAdcock) [inline src/css/embed.scss:322]: Definitely!

**Comment by SiAdcock** [comment]
Thanks Joe, I've added the comment on the 1px width. Merging.

**Pull request merged**